server/services/lduService.js

```diff
@@ -13,19 +14,27 @@ const logger = require('../../log')
 module.exports = function createLduService(deliusClient, activeLduClient) {
   return {
     async getAllProbationAreas() {
-      return deliusClient.getAllProbationAreas()
+      const { content: probationAreas } = await deliusClient.getAllProbationAreas()
+      probationAreas.sort((a, b) => a.description.localeCompare(b.description, 'en', { ignorePunctuation: true }))
+      return probationAreas
     },
 
     async getLdusForProbationArea(probationAreaCode) {
```

Comment:
The return type doesn't make sense based on the methods names.

A bit of modelling here would be good. 

We could say that the object we you are returning represents a probation area.
A probation area consists of a list of a description and a list of LDUs

```js
{
  description: 'some description',
  ldus: [
    ...  some ldus 
 ]
}
```